pyexchange/uniswapv2_analytics.py

```diff
@@ -150,6 +150,9 @@ def __init__(self, web3: Web3, token_config_path: str, keeper_address: Address,
             self._last_config = None
             self.token_config = self.get_token_config().token_config
 
+    def get_current_block(self) -> int:
+        return int(self.web3.eth.getBlock('latest')['number'])
```

Comment:
`web3.eth.blockNumber` is the usual way we do it in `pymaker`.

Reply:
Is there an advantage to using one over another?